src/libultra/io/vi.c

```diff
@@ -1,13 +1,13 @@
 #include "global.h"
 
-__OSViContext D_80098060[2] = { 0 };
-__OSViContext* __osViCurr = &D_80098060[0];
-__OSViContext* __osViNext = &D_80098060[1];
+__OSViContext vi[2] = { 0 };
```

Comment:
Without this being static, will a generic name `vi` cause any issues with name conflicts?